cluster/mesos/cluster_test.go

```diff
@@ -66,11 +66,6 @@ func TestContainerLookup(t *testing.T) {
 	// Container engine/name matching.
 	assert.NotNil(t, c.Container("test-engine/container1-name1"))
 	assert.NotNil(t, c.Container("test-engine/container1-name2"))
-	// Swarm ID lookup.
-	assert.NotNil(t, c.Container("swarm1-id"))
-	// Swarm ID prefix lookup.
-	assert.NotNil(t, c.Container("swarm1-"))
-	assert.Nil(t, c.Container("swarm"))
 	// Match name before ID prefix
 	cc := c.Container("con")
 	assert.NotNil(t, cc)
```

Comment:
We don't really test `com.docker.swarm.mesos.task` currently. I just pushed an update to the test.
We can leave out `com.docker.swarm.mesos.name` as it's not a mandatory field.